ExploreASL_Initialize.m

```diff
@@ -190,7 +190,7 @@ function xASL_init_PrintLogo()
     '                    ## |                                                                         \n'...
```

Comment:
The changes here in `ExploreASL_Initialize` have nothing to do with `xASL_im_ResampleLinearFair` right, I would move them to another issue

Reply:
Can't you just review them here? It was really only cosmetics and not worth a separate branch.